### ISSUE TITLE
[bug]: [4.0.0rc1] using TAESD as VAE throws error

### ISSUE THREAD

**Issue by grunblatt-git** [invoke-ai/InvokeAI#5937]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

macOS

### GPU vendor

Apple Silicon (MPS)

### GPU VRAM

24

### Version number

4.0.0rc1

### Browser

Chrome

### What happened

When selecting TAESD (https://huggingface.co/madebyollin/taesd) as VAE Invoke throws the following error after image generation (before saving the image to the gallery).

```
[InvokeAI]::ERROR --> Traceback (most recent call last):
  File "invokeai/.venv/lib/python3.10/site-packages/invokeai/app/services/session_processor/session_processor_default.py", line 160, in _process
    outputs = self._invocation.invoke_internal(
  File "invokeai/.venv/lib/python3.10/site-packages/invokeai/app/invocations/baseinvocation.py", line 289, in invoke_internal
    return self.invoke(context)
  File "invokeai/.venv/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "invokeai/.venv/lib/python3.10/site-packages/invokeai/app/invocations/latent.py", line 840, in invoke
    assert isinstance(vae_info.model, (UNet2DConditionModel, AutoencoderKL))
AssertionError
```

### What you expected to happen

Images can be generated with TAESD

### How to reproduce the problem

(Re)install TAESD by pasting importing `madebyollin/taesd` from the model editor. Choose TAESD as VAE. 
Optionally set VAE Precision to FP16 (this was required in Invoke v3.x.x to use taesd)

### Additional context

Uninstalling and reinstalling taesd did not change the behavior

**Comment by Harvester62** [comment]
This issue has been fixed in main development branch and in the upcoming v4.0.0: https://github.com/invoke-ai/InvokeAI/pull/5936

**Comment by grunblatt-git** [comment]
fixed in 4.0.0rc2
